public/docs/ts/latest/guide/style-guide.jade

```diff
@@ -341,11 +340,7 @@ a(href="#toc") Back to top
 
 .s-rule.do
   :marked
-    **Do** use upper camel case for services.
-
-.s-rule.do
-  :marked
-    **Do** suffix services with `Service` when it is not clear what they are (e.g. when they are nouns).
+    **Do** suffix services with `Service` when it is not clear what they are, such as when they are nouns.
```

Comment:
>**Do** suffix a service class name with `Service`. For example, something that gets data or heroes should be called a `DataService` or a `HeroService`. 
>
>A few terms are unambiguously services. They typically indicate agency by ending in "_er_". You may prefer to name a service that logs messages `Logger` rather than `LoggerService`. Decide if this exception is agreeable in your project. As always, strive for consistency.